apps/indexer/lib/indexer/fetcher/celo_validator_group_votes.ex

```diff
@@ -0,0 +1,121 @@
+defmodule Indexer.Fetcher.CeloValidatorGroupVotes do
```

Comment:
I feel we have too many epoch related fetcher modules and we will need to refactor these into a single CeloEpochData fetcher at some point